File: snippets/rust/create_full_snapshot.rs
async fn main() -> Result<(), qdrant_client::QdrantError> {
    use qdrant_client::Qdrant;

    let client = Qdrant::from_url("http://localhost:6334").build()?;

    client.create_full_snapshot().await?;
    Ok(())
}

File: snippets/go/create_full_snapshot.go
package client

import (
	"context"
	"fmt"

	"github.com/qdrant/go-client/qdrant"
)

func createFullSnapshot() {
	client, err := qdrant.NewClient(&qdrant.Config{
		Host: "localhost",
		Port: 6334,
	})
	if err != nil {
		panic(err)
	}

	snapshot, err := client.CreateFullSnapshot(context.Background())
	if err != nil {
		panic(err)
	}
	fmt.Println("Snapshot created: ", snapshot.Name)
}
